support-frontend/cloud-formation/cfn.yaml

```diff
@@ -296,14 +306,52 @@ Resources:
       - TargetGroup
       - ElasticLoadBalancer
 
+  ReducedHealthyInstancesAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
+      AlarmName: !Join
+        - ' '
+        - - !FindInMap [ StageVariables, !Ref Stage, AlarmUrgent ]
+          - !Ref 'Stage'
+          - 'Reduced number of healthy instances for support-frontend'
+      AlarmDescription: !Join
+        - ' '
+        - - 'Impact - Imminent issue cannot sell any subscriptions or contributions products.'
+          - !FindInMap [ StageVariables, !Ref Stage, AlarmProcess ]
```

Comment:
given that all of these alarms are conditional on `CreateProdResources` (except `StateMachineUnavailableAlarm` for some reason, should it be?), I wonder if all the substitutions of `Stage` variables could be simplified somehow? Just write "prod" where appropriate? Is it a general rule that we never have alarms on CODE resources?

Reply:
see what you mean that's a good point.  In that case I should really put them in a separate Map that isn't realted to stages.  I actually just wanted a simple constant, and the only way to do that in cloudformation seems to be to use a Mappings 😥

Reply:
regarding alarms on CODE, I was thinking about that too, I was wondering if we should have exactly the same alarms but just going to a dummy mailbox.  This would make it a lot easier to test my changes and also minimise the differences between CODE and PROD.  In the end I decided not to address it at this point.

Reply:
@joelochlann I have made a Constants section, and made that other alarm PROD only as per your suggestions.